source/opt/fold.h

```diff
@@ -28,6 +28,8 @@ namespace opt {
 
 class InstructionFolder {
  public:
+  InstructionFolder(IRContext* context) : context_(context) {}
```

Comment:
Please make this an explicit constructor

Reply:
Done. Editing on github isn't too bad, heh.